src/maker/config.rs

```diff
@@ -12,14 +12,20 @@ use super::api::MIN_SWAP_AMOUNT;
 /// Maker Configuration, controlling various maker behavior.
 #[derive(Debug, Clone, PartialEq)]
 pub struct MakerConfig {
-    /// Network listening port
-    pub network_port: u16,
     /// RPC listening port
     pub rpc_port: u16,
     /// Minimum Coinswap amount
     pub min_swap_amount: u64,
+    /// target listening port
+    pub network_port: u16,
+    /// control port
+    pub control_port: u16,
     /// Socks port
     pub socks_port: u16,
+    /// Authentication password
+    pub tor_auth_password: String,
+    /// Onion hostname
+    pub hostname: String,
```

Comment:
now that we have auto fetching, do we need the hostname in config?

Reply:
the hostname can be stored in the in-memory maker struct, for easier access later on.

Reply:
Allowing user to change hostname manually in any way can only cause more confusion and mistakes.